converter.go

```diff
@@ -202,10 +197,7 @@ func stringToValue(dest *driver.Value, srcColumnMeta execResponseRowType, srcVal
 		if err != nil {
 			return err
 		}
-		tt := time.Unix(sec, nsec)
```

Comment:
This is a good catch. I think the old behavior was a wrong answer.

Reply:
Agree. Just verified it.